src/dev/src/app/custom-props/custom-props.demo.ts

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { Component } from '@angular/core';
+import { runCssVarsPolyfill } from '@clr/base/src/lib/utils/css-vars/css-vars';
```

Comment:
I think this is the same as above, if this doesn't work it's because it's probably missing from the `index.ts` and `public_api.ts` export. Again minor nit we can fix later.

`import { runCssVarsPolyfill } from '@clr/base';`